tests/hash/test_generic_hash.py

```diff
@@ -404,6 +404,7 @@ def test_nexthop_flap(rand_selected_dut, tbinfo, ptfhost, fine_params, mg_facts,
         flap_interfaces(rand_selected_dut, [interface], times=3)
         pytest_assert(wait_until(10, 2, 0, check_default_route, rand_selected_dut, uplink_interfaces.keys()),
                       'The default route is not restored after the flapping.')
+        time.sleep(5)
```

Comment:
I suggest to change the logic to rely more robust logic, for example check wether the routes were installed on the ASIC